docs/components/base/_index.md

```diff
@@ -12,201 +12,35 @@ imageAlt: "base"
 
 A *base* is the platform that the other parts of a mobile robot attach to.
 
-Configure your robot's *base* component to reference any *motor* components attached to the base.
 By configuring a base component, organizing individual components to produce coordinated movement, you gain an interface to control the movement of the whole physical base of the robot without needing to send separate commands to individual motors.
 
-<img src="../img/base/base-trk-rover-w-arm.png" alt="A robot comprised of a base (motors, wheels and chassis) as well as some other components. The wheels are highlighted to indicate that they are part of the concept of a 'base', while the non-base components are not highlighted. The width and circumference are required attributes when configuring a base component." />
+<img src="img/base-trk-rover-w-arm.png" alt="A robot comprised of a wheeled base (motors, wheels and chassis) as well as some other components. The wheels are highlighted to indicate that they are part of the concept of a 'base', while the non-base components are not highlighted. The width and circumference are required attributes when configuring a base component."/>
 
 Most mobile robots with a base need at least the following hardware:
 
-- A [board component](/components/board/) that can run a `viam-server` instance.
-   For example, a Raspberry Pi, or another model of single-board computer with GPIO (general purpose input/output) pins.
-
+- A [board](/components/board/).
 - Some sort of actuators to move the base.
   Usually [motors](/components/motor/) attached to wheels or propellers.
-
 - A power supply for the board.
-
 - A power supply for the actuators.
-
 - Some sort of chassis to hold everything together.
 
 Example wiring diagram for a base with one motor on each side:
```

Comment:
Consider moving the wiring diagram to the wheeled base page, with a note that it directly corresponds to the example JSON shown there. That way 1. people don't have to scroll past it before getting to the API and 2. it's on the same page as its matching JSON with the same pin numbers and everything

Reply:
See my pr note-- this image is helpful. Also see for info as to why boat page is draft! Could have saved you some time reviewing 😆

Reply:
I saw your note! I am speaking of the wiring diagram, not the base diagram. I don't think people need to see the wiring to understand what a base is, and I stand by the scrolling point and the matching JSON point.

When something is marked as a draft I still default to reviewing it when I'm not explicitly told not to, since there are a variety of reasons for making drafts and maybe it'll land soon! But I guess I could've inferred by the "skeleton" bit--sorry :)

Reply:
Oh yes, sorry for the misunderstanding! I thought you were referring to the first diagram. I'll move just the wiring diagram over, that makes sense